chart/templates/mayastor/agents/core/agent-core-and-ha-cluster-deployment.yaml

```
@@ -57,3 +57,34 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+        - name: agent-ha-cluster
+          resources:
+            limits:
+              cpu: {{ .Values.agents.ha.cluster.resources.limits.cpu | quote }}
+              memory: {{ .Values.agents.ha.cluster.resources.limits.memory | quote }}
+            requests:
+              cpu: {{ .Values.agents.ha.cluster.resources.requests.cpu | quote }}
+              memory: {{ .Values.agents.ha.cluster.resources.requests.memory | quote }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/mayastor-agent-ha-cluster:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - "-g=0.0.0.0:11500"
```

```suggestion
            - "--grpc-server-addr=0.0.0.0:50051"
```
nit: use long name of arguments to make it clearer.